studio/components/ui/Forms/FormHeader.tsx

```diff
@@ -1,8 +1,16 @@
 import ReactMarkdown from 'react-markdown'
 
-const FormHeader = ({ title, description }: { title: string; description?: string }) => {
+const FormHeader = ({
```

Comment:
this looks like we are not using flexbox properly 

@joshenlim

Reply:
what's the intended behaviour? do we want to add margins for all flexbox items?

Reply:
there shoudlnt be any margins in flex box

```
<div className="flex gap-3">
  <div>there is always..</div>
  <div>..."gap-3" space between us</div>
</div
```

Reply:
Can jump in later and fix. Or someone from front end do it.